Parser/lexer/lexer.c

```diff
@@ -660,17 +660,29 @@ tok_get_normal_mode(struct tok_state *tok, tokenizer_mode* current_tok, struct t
             else if (!(saw_r || saw_u) && (c == 'r' || c == 'R')) {
                 saw_r = 1;
             }
-            else if (!(saw_f || saw_b || saw_u || saw_t) && (c == 'f' || c == 'F')) {
+            else if (!(saw_f || saw_b || saw_u) && (c == 'f' || c == 'F')) {
                 saw_f = 1;
             }
-            else if (!(saw_t || saw_b || saw_u || saw_f) && (c == 't' || c == 'T')) {
+            else if (!(saw_t || saw_u) && (c == 't' || c == 'T')) {
                 saw_t = 1;
             }
             else {
                 break;
             }
             c = tok_nextc(tok);
             if (c == '"' || c == '\'') {
+                if (saw_b && saw_t) {
+                    return MAKE_TOKEN(_PyTokenizer_syntaxerror(
+                        tok,
+                        "can't use 't' prefix on bytes"));
```

Comment:
Let's be consistent with the error message. Also, it'd be ideal if we could use `_PyTokenizer_syntaxerror_known_range` so that the carets points to the prefix itself.

Reply:
What error message do you suggest here?

Reply:
Let's do `can't use '<first prefix>' and '<second prefix>' prefixes together` or something like that for all combinations?

Reply:
`can't use 'b' and 't' string prefixes at the same time`?

Reply:
"Together" sounds better than "at the same time" to me but I don't have a strong opinion.

Reply:
ok 👍